### DISCUSSION TITLE
After updating version 1.9.3 here, I fixed the LSPosed crash problem😍

### DISCUSSION THREAD

**Discussion by RainHoutas** [mywalkb/LSPosed_mod#147] [General]
THANK YOU VERY MUCH.
My LSPosed was die today.
I could not open zhe manager and I don't know what happend without log
I try all method I can thought include delete all modules one by one to try to konw if there is a module could not work normaly and reinstall magisk and lsposed,reopen the rigisk······
BUT NONE of them fixed my problems.
I try to get all information on internet and FINALLY I found this new version.
I upgraded with the attitude of giving it a try.
Then all work went back to normal.
THANK YOU!!!!!

**Comment by DanaTheScripter** [comment]
I had the same problem, I'm was trying to follow this [workaround](https://github.com/orgs/LSPosed/discussions/2602) but the workaround didn't work, it just basically don't start the LSPosed manager, but this LSPosed mod fixed the shell crash problem.

**Reply by RainHoutas** [reply]
Yes, I almost thought I would never use LSPosed anymore for my device🤞

**Reply by RainHoutas** [reply]
I've also tried switching back to Riru and it didn't work either.
